README.md

```diff
@@ -1,36 +1,47 @@
+
+Once you have created your instance, note down the 'Service Token' listed underneath 'Probes' in the box on the right of the screen. You will use this token to authenticate your local Prometheus with Weave Cloud.
 
-Cortex is an API compatible [Prometheus](https://prometheus.io)
-implementation, that natively supports multitenancy and scale-out clustering.
+Install the Cortex agent (an unmodified OSS Prometheus) on your Kubernetes cluster using the following command:
 
-Cortex is a [Weaveworks](https://weave.works) project that forms a major part
-of [Weave Cloud](https://cloud.weave.works). If you have any questions regarding
-our hosted Prometheus as a service, please
-contact [help@weave.works](mailto:help@weave.works).
+    $ kubectl -n kube-system apply -f 'https://cloud.weave.works/k8s/cortex.yaml?t=<token>'
 
-## Getting started
+Select the Cortex service in the Weave Cloud UI:
 
-Go to https://cloud.weave.works and sign up. Follow the steps to create
-a new instance.
+<p align="center"><img src="imgs/cloud.png" alt="Cropped screenshot of Weave Cloud showing Prometheus button as graph"></p>
 
-Once you have created your instance, note down the 'Service Token' listed
-underneath 'Probes' in the box on the right of the screen. You will use this
-token to authenticate your local Prometheus with Weave Cloud.
+Weave Cloud gives you some example queries that show you general system information.  You can of course [write your own Prometheus queries](https://prometheus.io/docs/querying/basics/).
 
-You can [download Prometheus from its website](https://prometheus.io/download/).
-You will need v1.2.1 or later.
 
-When you've got Prometheus, you will need
-to
-[configure it to discover your services](https://prometheus.io/docs/operating/configuration/) and
-also configure it to send its data to Weave Cloud by adding the following
-top-level stanza to `prometheus.yml`:
+## About Weave Cortex
+
+To learn more about Weave Cortex consult the follow documents / talks:
+
+- [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
+- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
+- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/weaveworks/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
+- Or join us on [our Slack channel](https://slack.weave.works) and ask us directly!
+
+## Using Cortex on other platforms
+
+Cortex can be used to monitor virtually any application on any platform, as the local agent is a specially configure, unmodified OSS Prometheus binary.  You can [download Prometheus from its website](https://prometheus.io/download/).  You will need v1.2.1 or later.
```

Comment:
Also, https://github.com/weaveworks/service-conf/pull/329 updated *our* retrievers to be 1.3.1. There's no justification on the PR, and I've forgotten whether that change was essential to that PR (which purported to fix links) or incidental. 

Which is all a rather long way of saying, should this be 1.3.1, not 1.2.1?